### PR TITLE
Update Conformance, Trademark

### PR DESCRIPTION
**Pull request by omerbensaadon**
Adding @salaboy and myself to Conformance Task Force Leads, adding myself as TM Committee member

# Changes

-
-
-

/kind <kind>

Fixes #

**Release Note**

```release-note

```

**Docs**

```docs

```

### CODE REVIEW

File: peribolos/knative.yaml [hunk 1]
```diff
@@ -736,10 +736,11 @@ orgs:
         repos:
           specs: write
         teams:
-          Conformance WG Leads:
-            description: The Working Group leads for Conformance
+          Conformance Task Force Leads:
+            description: The Task Forceleads for Conformance
```
Comment (rhuss) [inline peribolos/knative.yaml:740]:
```suggestion
            description: The Task Force leads for Conformance
```

**Comment by bsnchan** [comment]
@omerbensaadon could you update https://github.com/knative/community/blob/main/TRADEMARK-COMMITTEE.md as well please :)

**Comment by dprotaso** [comment]
I think you can't rename the working group while editing users in that group.

Probably do the rename in a separate PR

**Comment by rhuss** [comment]
I just added the new Task Force team manually to make Peribolos happy

**Pull request merged**